packages/type-storage/src/key/index.ts

```diff
@@ -13,17 +12,14 @@ import formatParams from './params';
 
 export type Keygen = (...keyParams: Array<Storage$Key$Value>) => Uint8Array;
 
-export default function bindKey <T> ({ isUnhashed, key, params }: SectionItem<T>): Keygen {
+export default function bindKey<T> ({ isUnhashed, key, params }: SectionItem<T>): Keygen {
   const prefix = u8aFromString(key);
 
   return (...keyParams: Array<Storage$Key$Value>): Uint8Array => {
-    const postfix = keyParams.length !== 0
-      ? u8aConcat.apply(null, formatParams(params, keyParams))
-      : new Uint8Array([]);
+    const postfix =
+      keyParams.length !== 0 ? u8aConcat.apply(null, formatParams(params, keyParams)) : new Uint8Array([]);
```

Comment:
The extra whitespace doesn't cost anything and reads a bit better, so suggesting -

```
const postfix = keyParams.length !== 0
  ? u8aConcat.apply(null, formatParams(params, keyParams))
  : new Uint8Array([]);
```

Just a bit easier to see option a & option b in one glance

Reply:
OK. While I agree here specifically, I also believe code formatting is subjective.

Small proposal: go prettier the whole way, 80 chars/line, and my code formatting === your code formatting everywhere.